scripts/setup-conf.sh

```diff
@@ -82,13 +75,19 @@ EOF
 echo "include 'streaming_replication.conf'" >> $CONF
 fi
 
-if [[  -f ${ROOT_CONF}/extra.conf ]];then
-    rm $CONF/extra.conf
-fi
+if [[ ! -f ${ROOT_CONF}/extra.conf ]]; then
+    # If it doesn't exists, copy from /settings directory if exists
+    if [[ -f /settings/extra.conf ]]; then
+      cp -f /settings/extra.conf ${ROOT_CONF}/extra.conf
+      echo "include 'extra.conf'" >> $CONF
+    else
+      # default value
+      if [[  ! -z $EXTRA_CONF ]]; then
```

Comment:
If I understand it correctly, `$EXTRA_CONF` might have whitespaces, so surround with double quote to prevent globbing. Also replace `! -z` with equivalent `-n` (true if non zero length).

```suggestion
      if [[  -n "$EXTRA_CONF" ]]; then
```